README.md

```diff
@@ -4,7 +4,8 @@
 
 ## Examples
 
-- [Basic](#basic)
+- [Basic (E2E)](#basic-e2e) testing
```

Comment:
I wonder whether we should refer to it as "default" rather than "basic":

`End-to-End (default)`

Reply:
@MikeMcC399
Thanks for your comments! It's always good to have another pair of eyes looking at things!

The name "Basic" is historical when there was only E2E available and this example was a basic E2E example. I suggest changing the title to simply "End-to-End" in the table of contents (no "default") and at the top of the section, and leaving the actual workflow with its current name "example-basic.yml".  

I wanted to avoid taking this further and renaming "basic" at the workflow yml level as well. That would have other knock-on effects to several other places, including to the basic-pnpm workflow, which then also ought to be renamed, and to scripts, etc.

I have committed the updates and you can use the links I posted to review the rendered Markdown.

I also added a badge for Component Testing which had been previously omitted.

Reply:
I will resolve this thread and let you comment on the changes if necessary.

Reply:
Thanks for the context! That seems reasonable